baseServer_test.go

```diff
@@ -53,6 +68,21 @@ func (tba *TestAgent) CreateTestMessage() TestMessage {
 	}
 }
 
+func CreateInfiniteLoopMessage() InfiniteLoopMessage {
+	return InfiniteLoopMessage{
+		basePlatformSOMAS.BaseMessage{},
+	}
+}
+
+func InfLoop() {
+	i := 0
+	for {
+		if i == -1 {
+			return
+		}
```

Comment:
since this will never happen, you can just remove it. was this to avoid the linting error? (warning, function will spin)?

Reply:
to trick the compiler into letting me make a infinite spinner